src/nft/components/collection/CollectionStats.tsx

```diff
@@ -143,8 +155,8 @@ const CollectionName = ({
           gap="8"
           height="32"
         >
-          {collectionStats.discordUrl ? (
-            <SocialsIcon href={collectionStats.discordUrl}>
+          {collectionStats.discordUrl || isCollectionStatsLoading ? (
+            <SocialsIcon href={collectionStats.discordUrl ?? ''}>
```

Comment:
Instead of adding these conditionals atomically and having some null check logic, we could do one check for isCollectionStatsLoading and map an array of length 4 to 4 divs with the loading class